osu.Game/Online/Spectator/SpectatorStreamingClient.cs

```diff
@@ -27,14 +23,18 @@
 
 namespace osu.Game.Online.Spectator
 {
-    public class SpectatorStreamingClient : Component, ISpectatorClient
+    public class SpectatorStreamingClient : CompositeDrawable, ISpectatorClient
     {
         /// <summary>
         /// The maximum milliseconds between frame bundle sends.
         /// </summary>
         public const double TIME_BETWEEN_SENDS = 200;
 
-        private HubConnection connection;
+        private readonly HubClientConnector connector;
+
+        protected virtual IBindable<bool> IsConnected => connector.IsConnected;
```

Comment:
This is not how bindables work. Make this non-virtual and bind to the connector's bindable instead.